### ISSUE TITLE
Add Lasso

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#72]
Add Lasso model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference Lasso Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/LassoRegression.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an Lasso  as a potential best fit model.

**Comment by Tanuj2552** [comment]
please assign me this issue. I will do the best possible

**Comment by Tanuj2552** [comment]
New PR #100 has been added to solve this issue.
